src/main/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BaseRecalibrator.java

```diff
@@ -66,23 +66,27 @@
  *
  * <h3>Examples</h3>
  * <pre>
- * java -Xmx4g -jar GenomeAnalysisTK.jar \
- *   -T BaseRecalibrator \
+ * ./gatk BaseRecalibrator \
```

Comment:
Update program group, see my comments to ApplyBQSRSpark and update the example command.

Reply:
Fixed